portfolio/.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
```

Comment:
nit: Add a .gitignore file to keep this out of the repo https://git-scm.com/docs/gitignore

Reply:
Added .eslintrc.js to the .gitignore file, thank you for the tip